Method scvis task dimred

michalk8 opened the pull request:
<!-- You can erase any parts of this template not applicable to your Pull Request. -->

### Submission type

* [ ] This submission adds a new dataset
* [x] This submission adds a new method
* [ ] This submission adds a new metric
* [ ] This submission adds a new task
* [ ] This submission adds a new Docker image
* [ ] This submission fixes a bug (link to related issue: )
* [ ] This submission adds a new feature not listed above

### Testing

* [x] This submission was written on a forked copy of SingleCellOpenProblems
* [x] GitHub Actions "Run Benchmark" tests are passing on this base branch of this pull request (include link to passed test: )
  https://github.com/michalk8/SingleCellOpenProblems/actions/runs/702704279
* [x] If this pull request is not ready for review (including passing the "Run Benchmark" tests), I will open this PR as a draft (click on the down arrow next to the "Create Pull Request" button)

### Submission guidelines

* [x] This submission follows the guidelines in our [Contributing](../blob/master/CONTRIBUTING.md) document
* [x] I have checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change

wes-lewis committed:
Create alra.py

Add alra.py file as created previously from the wrong branch.

wes-lewis committed:
Delete alra.py

removing from the incorrect task

wes-lewis committed:
Create alra.py

Add alra to denoising task

danielStrobl committed:
Update README.md

Correcting my name in README.md

LuckyMD committed:
Merge branch 'master' into danielStrobl-patch-1

dburkhardt committed:
Update CONTRIBUTING.md

michalk8 committed:
Add docker image

michalk8 committed:
Add skeleton method

michalk8 committed:
Implement method

michalk8 committed:
Fix some small bugs

michalk8 committed:
Add preprocessing

dburkhardt committed:
Dockerfile bugfix

dburkhardt committed:
Force docker_build on `master`

LuckyMD committed:
Merge pull request #245 from singlecellopenproblems/danielStrobl-patch-1

Update README.md

LuckyMD committed:
make var names unique for tenx data loader

LuckyMD committed:
Merge branch 'master' of github.com:singlecellopenproblems/SingleCellOpenProblems

wes-lewis committed:
Update alra.py

Change to utilize alra.R support script.

github-actions[bot] committed:
pre-commit

wes-lewis committed:
Create alra.R

Add support script alra.R

github-actions[bot] committed:
pre-commit

wes-lewis committed:
Merge branch 'master' into task-denoising-method-alra

wes-lewis committed:
Update alra.R

Changed comment to equal <80 characters in length

wes-lewis committed:
Update alra.py

Remove scran pooling and log-cpm imports and shorten comment

dburkhardt committed:
Merge pull request #297 from wes-lewis/task-denoising-method-alra

Add method ALRA to denoising task

lazappi commented:
Thanks @michalk8 🎉 ! This looks good to me. I'm just going to hold off merging methods until the pre-processing stuff is sorted out but I'll let you know if you need to do anything.

dburkhardt committed:
Update README.md

dburkhardt committed:
Update README.md

dburkhardt committed:
Update README.md

atong01 committed:
Add Method SCOT to Multi-modal data Integration Task (#298)

* Add SCOT method to multimodal data integration task

* pre-commit

* Fix flake things

* pre-commit

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

michalk8 committed:
Add trustworthiness score for dimred task (#258)

* Add trustworthiness score

* Address comments, add test

* Fix test

* Update README.md

* Update README.md

* Update README.md

* Update README.md

Co-authored-by: Daniel Burkhardt <burkhardt.d.b@gmail.com>

lazappi committed:
Dimred methods preprocessing (#301)

* Add preprocessing.py for dim red methods

This file should contain standard pre-processing functions used to
create variations on different methods

* Add pre-processing function existing dim red methods

Create a new PHATE variant using Scanpy pre-processing

* Update dim red README to describe pre-processing

* pre-commit

* Pre-commit fixes

* Move PHATE import inside function

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
Co-authored-by: Daniel Burkhardt <burkhardt.d.b@gmail.com>

dburkhardt committed:
Fix s3 bucket clash

dburkhardt committed:
set branch variable in S3 setup job

dburkhardt committed:
BRANCH -> WKDIR for s3

dburkhardt committed:
rm echo

dburkhardt committed:
use env.BRANCH

dburkhardt committed:
fix nf wkdir

dburkhardt committed:
fix region

lazappi commented:
@michalk8 Can probably replace the pre-processing here as well.

michalk8 committed:
Merge branch 'master' into method-scvis-task-dimred

michalk8 committed:
Add new preprocessing

michalk8 committed:
Fix preprocessing
